tests/tests-html-elements.php

```diff
@@ -135,7 +135,7 @@ public function test_category_dropdown() {
 	 */
 	public function test_year_dropdown() {
 		$current_year = date( 'Y' );
-		$expected     = '<select name="year" id="" class="edd-select " data-placeholder="">';
+		$expected     = '<select name="year" id="edd-year-dropdown_year" class="edd-select " data-placeholder="">';
```

Comment:
Because the html `select` method changes `-` to `_`, this needs to be changed to use `_` for the tests to pass the expected output (same for month).